metering/metering-installing-metering.adoc

```diff
@@ -16,7 +16,7 @@ include::modules/metering-install-operator.adoc[leveloffset=+1]
 // Including this content directly in the assembly because the workflow requires linking off to the config docs, and we don't current link
 // inside of modules - klamenzo 2019-09-23
 [id="metering-install-metering-stack_{context}"]
-== Installing the Metering stack
+== Installing the metering stack
```

Comment:
I wonder if we need to propagate this change from `Metering` to `metering` throughout the rest of this docset as a change down the road.

Reply:
I agree, I changed it on the install page to match that of "Upgrading metering."